### PR TITLE
THEMES-760: Identity Blocks | Login links block

### PR DESCRIPTION
**Pull request by vgalatro**
## Description

This PR adds in the Login links block that accompanies the sign up and log in forms. It also corrects some styling issues with the single column layout related to the display of these blocks.

## Jira Ticket

- [THEMES-760](https://arcpublishing.atlassian.net/browse/THEMES-760)

## Acceptance Criteria

Login links block is converted to v2.

## Test Steps

1. Checkout this branch `git checkout THEMES-760`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block,@wpmedia/single-column-layout-block`
3. Create a page using the single column layout.
4. Add the Identity Log In - Arc Block and the Identity Login Links - Arc Block to the "body" section of the page.
5. In the Identity Login Links - Arc Block make sure "Show Forgot Password link" and "Show Sign up link" are checked. You can customized the URLs for each as well.
6. Make sure the login links appear below the form as seen on the designs (https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?type=design&node-id=12874-178005&mode=design&t=JptOC1IZ0G9wCmAD-4).
7. Now replace Identity Log In - Arc Block with Identity Sign Up - Arc Block.
8. In the Identity Login Links - Arc Block make sure "Show Login link" is checked. You can customized the URL as well.
9. Make sure the login link appear below the form as seen on the designs (https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?type=design&node-id=8411-119293&mode=design&t=BeXCdtxm1ekWDLRJ-4).

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-760]: https://arcpublishing.atlassian.net/browse/THEMES-760?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: blocks/identity-block/intl.json [hunk 1]
```diff
@@ -276,7 +276,7 @@
 	},
 	"identity-block.login-links-signup": {
 		"de": "Sign up for an account.",
-		"en": "Sign up for an account.",
+		"en": "Don't have an account? Sign up",
```
Comment (nschubach) [inline blocks/identity-block/intl.json:279]: Can we update the others as well so we get proper translations whenever that's issued?

Comment (vgalatro) [reply blocks/identity-block/intl.json:279]: I tried, but for some reason when I download the files from localize it keeps the old values for the other languages. I don't know if I'm missing a setting when importing/exporting. I had the `Replace with base language` option checked, but it seems to not see the update.

Comment (nschubach) [reply blocks/identity-block/intl.json:279]: interesting... if you are going to add in the error test, I'll check this when you are done with that,  Overall it looks ok to me atm.

Comment (vgalatro) [reply blocks/identity-block/intl.json:279]: The error test is needed for the log in and sign up forms, this one just outputs links so there should be no need for that.

Comment (vgalatro) [reply blocks/identity-block/intl.json:279]: @nschubach , see my note above. Is there anything else you think needs to be changed, or can this be approved?

**Review by nschubach — APPROVED** [review]

**Pull request merged**